Neos.Neos/Resources/Private/Templates/Module/Management/Workspaces/Show.html

```diff
@@ -69,9 +69,19 @@
 										</td>
 										<td class="neos-action">
 											<f:if condition="{canPublishToBaseWorkspace}">
-												<button form="postHelper" formaction="{f:uri.action(action: 'publishNode', arguments: {node: change.node.contextPath, selectedWorkspace: selectedWorkspace})}" type="submit" class="neos-button neos-button-success neos-pull-right" title="{neos:backend.translate(id: 'publish', source: 'Main', package: 'Neos.Neos')}" data-neos-toggle="tooltip">
-													<i class="fas fa-check icon-white"></i>
-												</button>
+												<f:if condition="{document.isNew}">
+													<f:then>
+														<div class="neos-button neos-disabled neos-button-primary neos-pull-right"
+																 title="{neos:backend.translate(id: 'cantPublishSingleNodeInNewPage', source: 'Main', package: 'Neos.Neos')}">
+															<i class="fas fa-check icon-white"></i>
+														</div>
+													</f:then>
+													<f:else>
+														<button form="postHelper" formaction="{f:uri.action(action: 'publishNode', arguments: {node: change.node.contextPath, selectedWorkspace: selectedWorkspace})}" type="submit" class="neos-button neos-button-primary neos-pull-right" title="{neos:backend.translate(id: 'publish', source: 'Main', package: 'Neos.Neos')}" data-neos-toggle="tooltip">
```

Comment:
the else part is equal to before, except `neos-button-success` was changed to `neos-button-primary`

Reply:
new

<img width="97" alt="image" src="https://github.com/neos/neos-development-collection/assets/85400359/9aa3eb1e-1f8a-4e26-8da5-990553b4b6d9">


vs old

<img width="108" alt="image" src="https://github.com/neos/neos-development-collection/assets/85400359/2bd4684e-89f8-4d7c-acef-47c778869365">


@pKallert was this design change intentional? Maybe because the publish button below is also blue?

<img width="251" alt="image" src="https://github.com/neos/neos-development-collection/assets/85400359/7185c90c-241f-456b-a916-cb911530c058">